qadence/blocks/primitive.py

```diff
@@ -454,3 +454,34 @@ def _block_title(self) -> str:
 
         s += rf" \[params: {params_str}]"
         return s if self.tag is None else (s + rf" \[tag: {self.tag}]")
+
+
+class ProjectorBlock(PrimitiveBlock):
+    """The abstract ProjectorBlock."""
+
+    name = "ProjectorBlock"
+
+    def __init__(
+        self,
+        ket: str,
+        bra: str,
```

Comment:
Is this supposed to be given as a bitstring: e.g. "0010"?

Reply:
Yes, indeed. I can add a description in the docstring.

Reply:
Done.